### PR TITLE
More col removal

### PR DESCRIPTION
**Pull request by squirrelo**
- Removes more columns for EBI release
- Allows full pulldown of all cols if wanted
- Fixes host_common_name

### CODE REVIEW

File: knimin/lib/data_access.py [hunk 1]
```diff
@@ -489,7 +490,7 @@ def _months_between_dates(self, d1, d2):
         # Calculate the number of 12-month periods between the years
         return (d2.year - d1.year) * 12 + (d2.month - d1.month)
 
-    def format_survey_data(self, md, external_surveys=None):  # noqa
+    def format_survey_data(self, md, external_surveys=None, full=False):  # noqa
```
Comment (mortonjt) [inline knimin/lib/data_access.py]: Would be nice to update the docs

**Comment by mortonjt** [comment]
One minor comment.  Otherwise, its ready to merge

File: knimin/lib/data_access.py (hunk 1 above)
Comment (squirrelo) [inline knimin/lib/data_access.py]: Again, this is code that will be refactored as soon as it works and we can add tests, so documentation at this point is moot.

**Comment by squirrelo** [comment]
Ready for merge

**Pull request merged**